lib/manager/gradle-wrapper/artifacts.ts

```diff
@@ -0,0 +1,82 @@
+import Git from 'simple-git/promise';
+import { resolve } from 'path';
+import * as fs from 'fs-extra';
+import { logger } from '../../logger';
+import { UpdateArtifact, UpdateArtifactsResult } from '../common';
+import { exec, ExecOptions } from '../../util/exec';
+import { readLocalFile } from '../../util/fs';
+import { platform } from '../../platform';
+import { VERSION_REGEX } from './search';
+import { gradleWrapperFileName, prepareGradleCommand } from '../gradle/index';
+
+async function addIfUpdated(
+  status: Git.StatusResult,
+  filePath: string
+): Promise<UpdateArtifactsResult | null> {
+  if (status.modified.includes(filePath)) {
+    return {
+      artifactError: null,
+      file: {
+        name: filePath,
+        contents: await readLocalFile(filePath),
+      },
+    };
+  }
+  return null;
+}
+
+export async function updateArtifacts({
+  packageFileName,
+  updatedDeps,
+  newPackageFileContent,
+  config,
+}: UpdateArtifact): Promise<UpdateArtifactsResult[] | null> {
+  try {
+    const projectDir = resolve(packageFileName, './../../../');
```

Comment:
How do we know that this is the project directory?

Reply:
```suggestion
```
`config.localDir` should be used, cause gradle wrapper should only be in project root?

Reply:
fixed in https://github.com/renovatebot/renovate/pull/5842/commits/d6aab5cafa8a7ab4f6e2226d6fc7cb260854cf7c